examples/proteus.linux.yaml

```diff
@@ -11,7 +11,7 @@ extends:
 parameters:
   pyver: '3.7'
   python_site_packages_rel: 'lib/python3.7/site-packages'
-  python_host_packages: '/usr/lib/python3.7/dist-packages'
+  python_host_packages: '/usr/lib/python3.7/site-packages'
```

Comment:
FYI, the host packages are in `/usr/lib/python3/dist-packages` on ubuntu 18.04.

Reply:
Thanks for the info, on Arch Linux where it is `site-packages`. We can switch back to `dist-packages` if preferable as I think most people on Linux use Ubuntu